include/bluetooth/att.h

```diff
@@ -59,12 +59,53 @@ extern "C" {
 #if defined(CONFIG_BT_TESTING)
 
 int bt_eatt_disconnect_one(struct bt_conn *conn);
-int bt_eatt_connect(struct bt_conn *conn, uint8_t num_channels);
 size_t bt_eatt_count(struct bt_conn *conn);
 
 #endif /* CONFIG_BT_TESTING */
+
+/** @brief Connect Enhanced ATT channels
+ *
+ * Sends a series of Credit Based Connection Requests to connect @p num_channels
+ * Enhanced ATT channels. The peer may have limited resources and fewer channels
+ * may be created.
+ *
+ * @param conn The connection to send the request on
+ * @param num_channels The number of Enhanced ATT beaeres to request.
```

Comment:
Is there any value range here? Can the application e.g. request 125 channels? 
I suppose we also have a local max number of channels?

Reply:
I can add a reference to `CONFIG_BT_EATT_MAX`.

Reply:
I can add a reference to `CONFIG_BT_EATT_MAX`.

Reply:
I can add a reference to `CONFIG_BT_EATT_MAX`.

Reply:
Please do :)